README.md

```diff
@@ -0,0 +1,75 @@
+# clay-sitemaps
+
+A suite a tools for generating sitemaps for Clay sites.
+
+Clay-sitemaps provides two "standard" sitemap middleware, one for text and one for XML, that can get you up and running quickly. It also provides utilities that can help you create your own sitemap logic.
+
+## Quick Start
+
+```
+const sitemaps = require('clay-sitemaps')(someAmphoraInstance);
+
+app
+  .get('/sitemap.txt', sitemaps.standardText())
+  .get('/sitemap.xml', sitemaps.standardXML({
+    engines: { // engines for rendering sitemap templates
+      handlebars: yourHandlebarsEngine
```

Comment:
Hmmm I'm pretty sure we're moving to a Handlebars only approach because we want all the re-rendering benefits. @nelsonpecora might chime in here, but if that's the case then you wouldn't need to specify anything